openmp/runtime/src/include/omp-tools.h.var

```diff
@@ -1408,6 +1408,12 @@ typedef ompt_record_ompt_t *(*ompt_get_record_ompt_t) (
   ompt_buffer_cursor_t current
 );
 
+#ifdef _WIN32
+__declspec(dllexport)
```

Comment:
I guess the reasoning for adding this is to make sure the symbol gets exported? Should we add a visibility attribute as well?

Reply:
I am not so sure if there is anything we can do more on Windows other than `dllexport`.
We may be able to add `__attribute__((visibility("default")))` for non-Windows systems though.

Reply:
Sorry for not being clear. That's what I meant.